cmd/blockchaincmd/deploy.go

```diff
@@ -464,6 +466,18 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 		return PrintSubnetInfo(blockchainName, true)
 	}
 
+	ux.Logger.PrintToUser("You can use your local machine as a bootstrap validator on the blockchain")
+	ux.Logger.PrintToUser("This means that you don't have to to set up a remote server on a cloud service (e.g. AWS / GCP) to be a validator on the blockchain.")
+
+	useLocalMachine, err := app.Prompt.CaptureYesNo("Do you want to use your local machine as a bootstrap validator?")
+	if err != nil {
+		return err
+	}
+
+	if useLocalMachine {
+		bootstrapEndpoints = []string{"http://127.0.0.1:9650"}
```

Comment:
this might be tricky to assume this endpoint as port can be not 9650 but rather dynamic 
cc @felipemadero

Reply:
maybe we should ask user for the endpoint and not yes/no for local machine.
we can hint that it can be 127.0.0.1:9650 for the local machine